### ISSUE TITLE
[1.12.2] Crash with JAOPCA, Wrap up and IC2

### ISSUE THREAD

**Issue by LukewarmYeti** [TheLMiffy1111/JAOPCA#35]
https://pastebin.com/xpXHVHP0

Paste bin here, line 1853 for the actual crash. 

Says "net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from WrapUp (wrapup)". Also "Caused by: java.lang.RuntimeException: The output ItemStack 1xtile.air@0 is invalid."

Essentially when I have IC2 installed (latest 1.12 build, tried older, still occurs) and I try to add JAOPCA (latest 20th Oct), the game crashes before launching. I can however have Wrap up and IC2 together without any issues (I understand it's a library mod so I can't cause too many issues). I have other mods installed as well, but the log seems to mention wrap up, IC2 recipes and JAOPCA's IC2 module, so I'm pretty sure one of these is the issue.

Thanks!

**Comment by TheLMiffy1111** [comment]
did you change anything in the config?

**Comment by LukewarmYeti** [comment]
Nope, just added it to the mods folder unmodified (if your talking about JAOPCA). Haven't changed anything with IC2s config either.

Edit: Deleting the JAOPCA & IC2.ini fixed the issue. Hm. Maybe I did change the config a while back during testing, then never deleted the config file when deleting the mod. Then when i re added the mod it was already there, modified, which caused conflicts. Unsure if that was what actually happened, but seems to be the most likely.

Edit 2: I'm an idiot and didn't have JAOPCA added when I tested that above. Yeh it still crashes when deleting the config. No idea what the issue is.
